### DISCUSSION TITLE
Send PS errors to Slack

### DISCUSSION THREAD

**Discussion by nielswhydonate** [planetscale/discussion#330] [Q&A] [answered]
Is there a way to send PS database errors to Slack?

**Comment by tbarn** [comment]
Not today, but in the future we would love to explore webhooks in PlanetScale. No promises of when that is though.

**Comment by tbarn** [comment, marked as answer]
If you get an error in a deploy request, you can now use webhooks to send a notification somewhere else: https://planetscale.com/blog/introducing-webhooks
